gef-ui/api-tests/common-tests/role-validation-api-tests.js

```diff
@@ -22,9 +22,9 @@ const withRoleValidationApiTests = ({
   makeRequestWithHeaders,
   whitelistedRoles,
   successCode,
-  successHeaders,
+  successHeaders = undefined,
   disableHappyPath = false, // TODO DTFS2-6697: remove and test happy paths.
-  redirectUrlForInvalidRoles,
+  redirectUrlForInvalidRoles = undefined,
```

Comment:
Why are these undefined now?

Reply:
The successHeaders is optional & since this is consumed in typescript it needs to be marked as such - this is the neatest method I'm aware of (I have modified `redirectUrlForInvalidRoles` to take a value now)

Reply:
The problem comes from gef-ui\api-tests\amendments\facility-value.api-test.ts being a typescript file importing this function - I've added jsdoc now instead to mark these as optional & give better type safety anyway